### PR TITLE
Aycabas 03 08 2022 oneproductivityhub aspnetcore msal2 provider

### PR DESCRIPTION
**Pull request by aycabas**
Updated the sample with Msal2 Provider to consume existing AAD in the Teams Toolkit for Visual Studio template.

### CODE REVIEW

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

File: graph-toolkit-productivity-hub-aspnetcore/OneProductivityHubAspNetCore/Components/Welcome.razor [hunk 1]
```diff
@@ -0,0 +1,83 @@
+﻿@using Microsoft.Extensions.Configuration
```
Comment (tecton) [inline graph-toolkit-productivity-hub-aspnetcore/OneProductivityHubAspNetCore/Components/Welcome.razor:1]: Are these components not used in this sample? It seems that `Tab.razor` uses mgt-* components.

Comment (aycabas) [reply graph-toolkit-productivity-hub-aspnetcore/OneProductivityHubAspNetCore/Components/Welcome.razor:1]: Components come with the default template so I didn't remove them. But yes I don't use the components in the project. Do you want me to clean up everything unused?

File: graph-toolkit-productivity-hub-aspnetcore/OneProductivityHubAspNetCore/wwwroot/blank-auth-end.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿<!DOCTYPE html>
```
Comment (SLdragon) [inline graph-toolkit-productivity-hub-aspnetcore/OneProductivityHubAspNetCore/wwwroot/blank-auth-end.html:1]: Do we really need this file? 
I remember that we do not need `blank-auth-end.html` page in vscode tab sso project, it used internally in msal library, and do not need a real page.

Comment (aycabas) [reply graph-toolkit-productivity-hub-aspnetcore/OneProductivityHubAspNetCore/wwwroot/blank-auth-end.html:1]: MSAL2 provider requires this file to handle sso for the project, it's used in _Host.cshtml file.

File: graph-toolkit-productivity-hub-aspnetcore/OneProductivityHubAspNetCore/Components/Welcome.razor (hunk 1 above)
Comment (aycabas) [reply graph-toolkit-productivity-hub-aspnetcore/OneProductivityHubAspNetCore/Components/Welcome.razor]: @tecton I removed all the files unused. Thanks for letting me know.

Comment (tecton) [reply graph-toolkit-productivity-hub-aspnetcore/OneProductivityHubAspNetCore/Components/Welcome.razor]: Thank you! That would be great.

**Review by tecton — APPROVED** [review]

**Review by SLdragon — APPROVED** [review]

**Pull request merged**